public/js/tracking_protection.js

```diff
@@ -3,8 +3,38 @@
 /* global ga */
 /* global _dntEnabled  */
 
+// L8-L29 courtesy of https://github.com/mozilla/send
 
-if(!_dntEnabled()) {
+	function browserName() {
```

Comment:
indentation nit-picking: since this function is declared outside the `if` check now, the declaration should have the indentation removed.